pkg/cmd/roachtest/registry/test_spec.go

```diff
@@ -84,10 +84,28 @@ type TestSpec struct {
 	// cannot be run with encryption enabled.
 	EncryptionSupport EncryptionSupport
 
+	// SkipPostValidations is a set of post-validations that should be skipped
+	// after the test completes. This is useful for tests that are known to be
+	// incompatible with some validations. By default, tests will run all
+	// validations.
+	SkipPostValidations PostValidation
+
 	// Run is the test function.
 	Run func(ctx context.Context, t test.Test, c cluster.Cluster)
 }
 
+// PostValidation is a type of post-validation that runs after a test completes.
+type PostValidation int
+
+const (
+	// PostValidationReplicaDivergence is a post-validation that checks for
```

Comment:
nit: those comments look unhelpful. they just expand the name into sentence. maybe we can just refer to underlying sql query that it performs? e.g. is a post validation that fails test if `crdb_internal.check_consistency` finds replicas in inconsistent state. same for `PostValidationInvalidDescriptors` and `crdb_internal. invalid_objects`.